redash/utils/configuration.py

```diff
@@ -24,6 +24,11 @@ def __init__(self, config, schema=None):
         self.set_schema(schema)
 
     def set_schema(self, schema):
+        if isinstance(schema, dict):
```

Comment:
This is not needed: `schema` is always a dict.

Reply:
I was getting a "`NoneType` has no attribute..." And that's why I put it 😬